src/httpdata.cpp

```diff
@@ -38,7 +38,7 @@ HttpResponse& HttpData::getResponse()
   return m_HttpResponse;
 }
 
-void HttpData::setResponse(const QJsonObject& json)
+void HttpData::setResponse(const QJsonDocument& json)
```

Comment:
I went ahead and tried against Qt 5.7 and this will throw up.  As a compromise, perhaps a MACRO can toggle between the types - however the cascading problem here will be that the object assignments will also be affected.

I am installing 5.9 now and I assume it'll run without a hitch, thoughts?

Reply:
So looks like we need to update test files :)

Reply:
Ah, sorry. I had trouble getting my local Qt version with MSVC17 to work in QtCreator. That is why I didn't build the tests.

I'll take a look at them right away.